gradle.properties

```diff
@@ -39,6 +39,9 @@ ksp.useKSP2=true
 # https://github.com/JetBrains/intellij-platform-gradle-plugin/blob/33af9bdd1d0d8b996b911781ab23013dd14997f4/src/main/kotlin/org/jetbrains/intellij/platform/gradle/GradleProperties.kt#L104C12-L104C27
 org.jetbrains.intellij.platform.SelfUpdateCheck=false
 
+# We can now run ./gradlew runLocalIde on Android Studio, so we have test more comprehensively with Skate
+intellijPlatformTesting.idePath=/Applications/Android Studio.app/Contents
```

Comment:
talked offline but let's make this something developers specify locally in their home `~/.gradle/gradle.properties` and document it in the skate docs